pkg/create/templates/docker.go

```diff
@@ -3,38 +3,56 @@ package templates
 // Dockerfile is a default Dockerfile laid down by s2i create
 const Dockerfile = `
 # {{.ImageName}}
-FROM openshift/base-centos7
```

Comment:
I do understand your guys thoughts on this, but, I think that it is more useful for the user to get a fully working application when they use create, something where the make, and make test both work correctly and pass.  I had also considered removing the example that is provided in the repo, but instead provided a copy of what s2i create provides that users can browse through.